clap_generate/examples/bash_completion.rs

```diff
@@ -0,0 +1,11 @@
+use clap::App;
+use clap_generate::{generate, generators::Bash};
+use std::io;
+
+fn main() {
+    let mut app = App::new("myapp")
+        .subcommand(App::new("test").subcommand(App::new("config")))
+        .subcommand(App::new("hello"));
+
+    generate::<Bash, _>(&mut app, "myapp", &mut io::stdout());
```

Comment:
Just doing `generate::<Bash>(...)` is not working. I have to give `_`. Wonder why that is.

Reply:
The same reason you can't `HashMap::<String>::new()`